support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouSocialShare.jsx

```diff
@@ -43,6 +45,10 @@ const ContributionThankYouSocialShare = ({
   createReferralCodes,
   campaignCode,
 }: ContributionThankYouSocialShareProps) => {
+  useEffect(() => {
+    trackComponentLoad(OPHAN_COMPONENT_ID_SOCIAL);
+  }, []);
+
```

Comment:
I it's best to just send off one unified "social" event rather than one for each!